### PR TITLE
[Improment]Reset workload group's default value

### PR DESCRIPTION
**Pull request by wangbo**
### What problem does this PR solve?
1 set  cpu_share and memory_limit's default value to -1.
2 unify error message when workload group's value is invalid.

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [x] Yes. 

- Does this need documentation?
    - [ ] No.
    - [x] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

File: be/src/runtime/workload_group/workload_group.cpp [hunk 1]
```diff
@@ -345,19 +345,19 @@ WorkloadGroupInfo WorkloadGroupInfo::parse_topic_info(
 
     // 4 cpu_share
     uint64_t cpu_share = CgroupCpuCtl::cpu_soft_limit_default_value();
-    if (tworkload_group_info.__isset.cpu_share) {
+    if (tworkload_group_info.__isset.cpu_share && tworkload_group_info.cpu_share > 0) {
```
Comment (yiguolei) [inline be/src/runtime/workload_group/workload_group.cpp:348]: 如果我先从-1 设置为10，然后又想设置为-1 咋办？

Comment (wangbo) [reply be/src/runtime/workload_group/workload_group.cpp:348]: 不影响啊，fe的-1可以随便设置。传到be的时候，如果发现是-1，就使用be的默认值(cpu_soft_limit_default_value)啊

**Review by yiguolei — APPROVED** [review]

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**